lib/deepstream/list.rb

```diff
@@ -8,7 +8,12 @@ def initialize(*args)
     end
 
     def add(record_name)
-      set(@data << record_name) unless @data.include?(record_name)
+      unless @data.include?(record_name)
```

Comment:
'Lists and record names have an n:m relationship. A record name can be part of many lists and a list can contain many record names. A list can also contain the same record name multiple times'
https://deepstream.io/docs/client-js/datasync-list/